src/main/flight/gps_rescue.c

```diff
@@ -731,14 +737,17 @@ void gpsRescueUpdate(void)
                 // attempted initiation within 5m of home, and 'on the ground' -> instant disarm, for safety reasons
                 rescueState.phase = RESCUE_ABORT;
             } else {
-                // Otherwise, attempted initiation inside minimum activation distance, at any height -> landing mode
-                rescueState.intent.altitudeStep = -rescueState.sensor.altitudeDataIntervalSeconds * gpsRescueConfig()->descendRate;
-                rescueState.intent.targetVelocityCmS = 0; // zero forward velocity
-                rescueState.intent.pitchAngleLimitDeg = 0; // flat on pitch
-                rescueState.intent.rollAngleLimitDeg = 0.0f; // flat on roll also
-                rescueState.intent.proximityToLandingArea = 0.0f; // force velocity iTerm to zero
-                rescueState.intent.targetAltitudeCm = rescueState.sensor.currentAltitudeCm + rescueState.intent.altitudeStep;
-                rescueState.phase = RESCUE_LANDING;
+                // Otherwise, attempted the Rescue initiation inside the minimum activation distance
+                // don't climb, yaw quickly, enter descend mode and land
+                rescueState.intent.yawAttenuator = 1.0f; // yaw quickly to point to home
+                rescueState.intent.pitchAngleLimitDeg = 15.0f; // limited ability to pitch and roll
+                rescueState.intent.targetAltitudeCm = rescueState.sensor.currentAltitudeCm + (0.5f * rescueState.intent.targetLandingAltitudeCm); // climb 1m, in practice, just a little throttle blip
+                // climb a meter or two
```

Comment:
What line is this comment referring to? 
Sometimes the comment after line is confusing, but obvious from context. Not in this case. 

It it so annoying in code review..

Reply:
Previously I recall your view was that a comment should apply to the preceding line, generally.
In this case the comment referred to the previous line, where the target altitude was set to the current altitude plus a meter or so.

Reply:
Exactly the opposite. Comments on separate line shall IMO precede code being commented. Only exception is when comment is describing state after performing action on previous line (` // now we are ready`).
Another option is to start comment after statement and continue on next line, indented to first one
```
    int x = 10;  // This is a long inline comment that needs to be split
                 // into multiple lines for better readability
```

I can't recall any project where comment on following line are used.

It is very difficult for me to read code commented in this way. Every such comment forces me to re-read enclosing lines.